app/controllers/studies_controller.rb

```diff
@@ -112,7 +112,8 @@ def sync_study
       firecloud_permissions = Study.firecloud_client.get_workspace_acl(@study.firecloud_project, @study.firecloud_workspace)
       firecloud_permissions['acl'].each do |user, permissions|
         # skip project owner permissions, they aren't relevant in this context
-        if permissions['accessLevel'] =~ /OWNER/i
+        # also skip the readonly service account
+        if permissions['accessLevel'] =~ /OWNER/i || (Study.read_only_firecloud_client.present? && user == Study.read_only_firecloud_client.isser)
```

Comment:
Agreed... we can probably make it a [concern](https://api.rubyonrails.org/classes/ActiveSupport/Concern.html) or just a standard Ruby module.  Just want to make sure that isn't going to impact response handling, since the regular controllers all respond with HTML or JS, and the API is pure JSON.